addons/mail/static/src/models/activity_box_view.js

```diff
@@ -5,6 +5,8 @@ import { attr, many, one } from '@mail/model/model_field';
 
 registerModel({
     name: 'ActivityBoxView',
+    template: 'mail.ActivityBoxView',
+    templateGetter: 'activityBoxView',
```

Comment:
Having to define this `templateGetter` thing everywhere is tedious and not really useful if you ask me. I believe that it would be better to have one single consistent getter used across all templates, for instance `record`, so that we don't have to name it by hand.

Note that the name "templateGetter" is not really clear either. I don't think I would have understood what it meant if I didn't already know.

Reply:
We do this to reduce diff as much as we can. It's this or we must make lots of diff in all component templates, which is annoying.

So this `templateGetter` is best trade-off for now.